portfolio/src/main/webapp/script.js

```diff
@@ -20,7 +20,7 @@ function addRandomGreeting() {
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+const greeting = greetings[Math.floor(Math.random() * Math.floor(greetings.length))];
```

Comment:
Small style nitpick: It seems the indentation level of this line changed so it doesn't match the other lines near it, which I'm assuming wasn't intentional. If it looks even with the adjacent lines in your editor, check your tabs vs spaces settings (I can help if you're not sure what I'm referring to, idk if I'm being coherent rn or not).